packages/sit-onyx/src/components/OnyxTab/OnyxTab.vue

```diff
@@ -25,6 +25,7 @@ defineSlots<{
 const { densityClass } = useDensity(props);
 const tabsContext = inject(TABS_INJECTION_KEY);
 const skeleton = useSkeletonContext(props);
+const sizeClass = computed(() => `onyx-tab--${tabsContext?.size.value}`);
```

Comment:
Wouldn't it be enough to set the size class on `.onyx-tabs__tablist` element?

Reply:
In theory yes but I wanted to separate this since the font styles are responsibility of the tab and not of the parent (in my opinion) :)